ios/app/src/AppDelegate.m

```diff
@@ -50,7 +50,7 @@ -             (BOOL)application:(UIApplication *)application
         // Apple rejected our app because they claim requiring a
         // Dropbox account for recording is not acceptable.
 #if DEBUG
-        [builder setFeatureFlag:@"ios.recording.enabled" withBoolean:YES];
+        [builder setFeatureFlag:@"recording.enabled" withBoolean:YES];
```

Comment:
Please don't change this, it would be a backwards incompatible change, and as I'll get into later, it shouldn't be necessary at all.

Reply:
Reverted. Thanks